rgw/v2/tests/s3_swift/multisite_configs/test_sse_s3_bucket_enc_multipart_download_remote_site.yaml

```diff
@@ -0,0 +1,20 @@
+#polarian CEPH-11357, bug-2162337
+#script- test_sse_s3_kms_with_vault.py
+config:
+ user_count: 1
+ encryption_keys: s3
+ bucket_count: 1
```

Comment:
@viduship , do we need "remote_zone: archive" here as well

Reply:
@mkasturi18 , we do not. For active zones it picks up the remote site based on the previous methods already in ceph-qe-scripts.